packages/plugin-devtools/test/devtools.test.ts

```diff
@@ -0,0 +1,735 @@
+                '</table>';
+            expect(aboutMe.outerHTML).to.equal(aboutMeResult);
+
+            const family = devtools.querySelector('devtools-info .properties table:last-child');
+            const familyResult =
+                '<table>' +
+                '<tbody>' +
+                '<tr><td>index</td><td>1</td></tr>' +
+                '<tr><td>parent</td><td>ROOT</td></tr>' +
+                '<tr><td>siblings</td><td><ol style="list-style-type: none"><li> previous: HEADING1</li><li> next: PARAGRAPH</li></ol></td></tr>' +
+                '<tr><td>children</td><td><ol><li>CHAR</li><li>CHAR</li><li>CHAR</li></ol></td></tr>' +
+                '</tbody>' +
+                '</table>';
+            expect(family.outerHTML).to.equal(familyResult);
+        });
+        it('should select a sibling node with arrow (up, down)', async () => {
+            const node = devtools.querySelector(
+                'devtools-node .children devtools-node.element:nth-child(2)',
+            );
+            const span = node.firstElementChild;
+
+            node.dispatchEvent(
+                new KeyboardEvent('keydown', {
+                    key: 'ArrowDown',
+                    code: 'ArrowDown',
+                    bubbles: true,
+                }),
+            );
+            await longNextTick();
+            expect(span.classList.contains('selected')).to.equal(false, 'unselect');
+            const next = node.nextElementSibling;
+            expect(next.firstElementChild.classList.contains('selected')).to.equal(
+                true,
+                'select the next',
+            );
+
+            node.dispatchEvent(
+                new KeyboardEvent('keydown', {
+                    key: 'ArrowUp',
+                    code: 'ArrowUp',
+                    bubbles: true,
+                }),
+            );
+            await longNextTick();
+            expect(span.classList.contains('selected')).to.equal(true, 'select');
+            expect(next.firstElementChild.classList.contains('selected')).to.equal(
+                false,
+                'unselect the next',
+            );
+        });
+        it('should select a sibling node with arrow (left, right)', async () => {
+            const node = devtools.querySelector(
+                'devtools-node .children devtools-node.element:nth-child(2)',
+            );
+            const span = node.firstElementChild;
+
+            node.dispatchEvent(
+                new KeyboardEvent('keydown', {
+                    key: 'ArrowRight',
+                    code: 'ArrowRight',
+                    bubbles: true,
+                }),
+            );
+            await longNextTick();
+            expect(span.classList.contains('selected')).to.equal(false, 'unselect');
+            const next = node.nextElementSibling;
+            expect(next.firstElementChild.classList.contains('selected')).to.equal(
+                true,
+                'select the next',
+            );
```

Comment:
You can do `to.be.true` and the label at the end is not really useful IMO. This applies to a lot of lines... :p